lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -15,7 +15,7 @@
   },
```

Comment:
<p>To volitelné stahování je asi zbytečné - protože distribuujeme materiály přímo s tím. Ale zároveň to ani nepřekáží ;-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/pyladies-kurz/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Já právě nevěděl, jestli budem podporovat workflow, kdy si někdo stáhne ipynb z webu (což dřív AFAIK fungovalo) a pustí ho - tak jsem chtěl, aby to fungovalo i v tom případě.</p>